00-elements/035-magician.frag

```diff
@@ -11,13 +11,7 @@ uniform vec2 u_resolution;
 #include "../lib/stroke.glsl"
 #include "../lib/circleSDF.glsl"
 #include "../lib/flip.glsl"
-
-//GLOBAL_START
-vec3 bridge(vec3 c,float d,float s,float w) {
-    c *= 1.-stroke(d,s,w*2.);
-    return c + stroke(d,s,w);
-}
-//GLOBAL_END
```

Comment:
Please don't replace this, the `// GLOBAL_START` and `// GLOBAL_END` are use to expose the code to the script that compile the master plate for the printer

Reply:
Where should the `// GLOBAL_START/END` comments should go since the `bridge` function is now replaced by `#include "../lib/bridge.glsl"`?
Thank you

Reply:
That adds the license and #define flags to it. But I guess is ok, because the script for making the printing plate is in another repo. Thanks for fixing it. 
Side note. All this functions are now part of https://lygia.xyz/draw  and https://lygia.xyz/sdf

Reply:
Thank you Patricio!
Side note: Lygia is awesome, thank you again for this fantastic tool

Reply:
Thank you @MAKIO135 !!!